app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/securities/BillOfDischargeController.scala

```diff
@@ -40,28 +42,42 @@ class BillOfDischargeController @Inject() (
 )(implicit ec: ExecutionContext, viewConfig: ViewConfig)
     extends SecuritiesJourneyBaseController {
 
-  private val form: Form[YesNo] = selectBillOfDischargeForm
+  def showBOD3: Action[AnyContent] = show(BOD3)
```

Comment:
this can be all vals

Reply:
I usually leave them as `def` because in IntelliJ, you get a nice little icon that takes you to the routes entry which is convenient.  
<img width="523" alt="Screenshot 2022-08-11 at 10 05 58" src="https://user-images.githubusercontent.com/1114992/184101032-0fe43d1f-8ee9-43a1-b2ef-4b6a364c90b8.png">

Is there any disadvantage to having it as a `def` instead asides spawning a new object each time it's called? I'm happy to change if it will cause significant performance issues..

Reply:
I presume there is a performance and heap penalty, so I would prefer to keep it `final val` if possible